src/Core/src/Handlers/Editor/EditorHandler.Android.cs

```diff
@@ -48,6 +50,13 @@ protected override void SetupDefaults(AppCompatEditText nativeView)
 
 			DefaultTextColors = nativeView.TextColors;
 			DefaultPlaceholderTextColors = nativeView.HintTextColors;
+			DefaultBackground = nativeView.Background;
```

Comment:
Could we add a handler test in these cases with custom code to assign the Background?